### PR TITLE
Add Icelandic translation

### PR DESCRIPTION
**Pull request by osk**

### CODE REVIEW

**Comment by CoralineAda** [comment]
This is great! I'll put out a call for a native speaker to review the translation right now.

**Comment by osk** [comment]
Oops, missing description, but there are some words I'm not a 100% sure on and would be good to get confirmation on:

* þjóðerni → ethnicity
* kynvitund og tjáningar → gender identity and expression
* ríkisfang  → nationality
* kyneinkenna og kynhneigðar → sexual identiy and orientation
* raunheimum → offline

Missing translations for:

* trolling
* commit

**Comment by hph** [comment]
@osk `þátttakandi/þátttakendur` should always be spelled with three Ts and I think it should be `þátttakanda` in `hvern þann þáttakenda` (take note of the `e`, it's not used in the singular form). There's also some redundant/incorrect repetition in `Verkefnateymi er skylt til að halda trúnað er bundið trúnaði við þann sem tilkynnir atvik.` I may very well have missed something else but otherwise it looks good to me!

File: content/version/1/4/code-of-conduct.is.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+Dæmi um hegðun sem stuðlar að jákvæðu umhverfi:
+
+* Notkun á opnu og aðlaðandi tungumáli
+* Virðing fyrir mismunandi sjónarhornum og reynslu
+* Taka vel á móti uppbyggjandi gagnrýni
+* Leggja áherslu á það sem er best fyrir samfélagið
+* Sýna öðrum meðlimum samkennd
+
+Dæmi um óviðunandi hegðun þáttakenda:
```
Comment (hph) [inline content/version/1/4/code-of-conduct.is.md]: It should be `þátttakenda` (or `þátttakanda` if in singular).

File: content/version/1/4/code-of-conduct.is.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+## Ábyrgð okkar
+
+Umsjónarfólk verkefnisins er ábyrgt fyrir því að setja fram skýrar kröfur um
+ásættanlegra hegðun og gert er ráð fyrir að það gerðar séu viðeigandi og
+sanngjarnar úrbætur í svari við öllum tilvikum óviðunandi hegðunnar.
+
+Umsjónarfólk verkefnis hefur rétt og ber skylda til að fjarlægja, breyta, eða
+hafna athugasemdum, „commitum“, kóða, wiki breytingum, málum, og öðrum framlögum
+sem ekki eru í takt við þessar hátternisreglur, eða að banna tímabundið eða
+varanlega hvern þann þáttakenda fyrir hverja þá hegðun sem það telur
```
Comment (hph) [inline content/version/1/4/code-of-conduct.is.md]: `þátttakanda` (note the triple `t` and `a` over the `e`)

File: content/version/1/4/code-of-conduct.is.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+hvort sem það er á netinu eða í raunheimum. Það að vera fulltrúi verkefnis getur
+verið frekar skilgreint og skýrt af umsjónarfólki verkefnis.
+
+## Fullnusta
+
+Tilvik af ofbeldisfullri, áreitni, eða annarskonar óviðunandi hegðunnar geta
+verið tilkynntar með því að hafa samband við verkefnateymi í [NETFANG HÉR].
+Allar kvartanir verða yfirfarnar og rannskaðar og munu leiða til viðbragða sem
+eru talin nauðsynleg og viðeigandi fyrir aðstæður. Verkefnateymi er skylt til að
+halda trúnað er bundið trúnaði við þann sem tilkynnir atvik. Frekari upplýsingar
```
Comment (hph) [inline content/version/1/4/code-of-conduct.is.md]: `skylt til að halda trúnað er bundið trúnaði` seems redundant/wrong

Comment (osk) [reply content/version/1/4/code-of-conduct.is.md]: Yeah, that's me not cleaning up 😆

**Comment by osk** [comment]
Thanks @hph ✨

**Comment by hph** [comment]
Commits could be omitted (the use of `kóða`, or `code`, is enough) and trolling can be replaced by a few words with the same meaning (i.e., baiting people into arguments) and `(e. „trolling“)` right after it.

**Comment by benediktvaldez** [comment]
@hph I agree with that approach on `trolling`. There have been attempts to translate 

Although I'm not sure about omitting `commits`. As in, the meaning of the text wouldn't be the same, since commits and code aren't the same thing (commit messages vs the actual code being committed). Not sure how to handle it though. Perhaps it's ok to use the english word in that context, should be clear what is meant?

**Comment by osk** [comment]
@CoralineAda, as per the discussion above, how have issues like these been addressed in the past? Can we localize part of the text about trolling and leave a word that has no translation in quotes?

**Comment by CoralineAda** [comment]
I think that would be fine, yes.

**Comment by osk** [comment]
I've updated the wording slightly, think it works re trolling sentence, whatya think @hph

**Comment by osk** [comment]
@CoralineAda what can I do to push this forward?

@hph, have you looked at the changes re trolling?

**Comment by hph** [comment]
@osk I'd personally use the `(e. something)` form, but I'm not sure it's required. Otherwise it looks good to me :+1:

**Comment by CoralineAda** [comment]
This looks great, thank you so much!

**Pull request merged**